Gemfile

```diff
@@ -25,3 +25,5 @@ gem 'rspec-puppet',                                              '~> 5.0'
 gem 'rubocop',                                                   '~> 1.75'
 gem 'unf_ext',                                                   '~> 0.0.9'
 gem 'yamllint',                                                  '~> 0.0.9'
+gem 'rexml'
```

Comment:
Might be a good idea to at least pin the major version here, like we've done with the others: `~> 3.2`